CoreDisplay-patcher.command

```diff
@@ -52,8 +57,17 @@ oToolCoreDisplayPatched=(
   11882f5e04a525da6701777c814c920a '10.13.4 (17E199)'
   b5dd02fe05903d7c39791afc642f9b2b '10.13.5 (17F77)'
   2d71736504e14882b2eb5e994859ec09 '10.14.2 (18C54)'
+  9581c3d50658882e79325bcf5f510245 '10.14.5 (18F132)'
+  d4f2aca73c7915c639dbe9a7b9cf5ecd '10.14.6 (18G84)'
+  450241a06a439cd226c9e9cf5cf634d7 '10.14.6 (18G95, 18G1012 and 18G2022)'
 )
 
+# when I ran "./CoreDisplay-patcher.command" this was the post-output of "./CoreDisplay-patcher.command md5"
+# otool CoreDisplay: 450241a06a439cd226c9e9cf5cf634d7
```

Comment:
I can confirm the same hash and value:

```
otool -t CoreDisplay2 |tail -n +2 |md5 -q
450241a06a439cd226c9e9cf5cf634d7
```

```
hexdump CoreDisplay  | grep -i "31 DB 90 90 90 90 90 90 E9"
00187c0 20 31 db 90 90 90 90 90 90 e9 8a 07 00 00 49 89
```

The current code on master appears to be wrong though as far as I can tell, see https://github.com/Floris497/mac-pixel-clock-patch-V2/issues/343

Are you sure your code signing is working properly?
I've found out that the script silently discards all code signing errors and potentially leaves you with a broken file.
What is your output from
`sudo codesign -f -s - $CoreDisplayLocation`

Reply:
`d4f2aca73c7915c639dbe9a7b9cf5ecd` is unlikely to be a proper value though, this is the same one I saw in #343

Reply:
I agree with this, I patched my PR